core/templates/components/oppia-angular-root.component.ts

```diff
@@ -464,6 +464,7 @@ import { TopicsAndSkillsDashboardBackendApiService } from 'domain/topics_and_ski
 import { TopicsAndSkillsDashboardPageService } from 'pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.service';
 import { TopicViewerBackendApiService } from
   'domain/topic_viewer/topic-viewer-backend-api.service';
+import { TranslationLanguageService } from 'pages/exploration-editor-page/translation-tab/services/translation-language.service';
```

Comment:
Changes to this file are no longer required. Can you remove them?

Reply:
done!